package.json

```diff
@@ -61,7 +62,7 @@
     "ts-node": "^10.9.1",
     "typescript": "^5.0.4"
   },
-  "dependencies": {
-    "@types/iobroker": "npm:@iobroker/types@^5.0.6"
+  "peerDependencies": {
+    "@iobroker/types": "^5.0.8"
```

Comment:
I think this needs to be in `devDependencies` too, so it gets installed in this repo when developing.